CONTRIBUTING.md

```diff
@@ -0,0 +1,227 @@
+
+   ```bash
+   $ git clone git@github.com:YourLogin/pydicom.git
+   $ cd pydicom
+   ```
+
+3. Create a ``feature`` branch to hold your development changes:
+
+   ```bash
+   $ git checkout -b my-feature
+   ```
+
+   Always use a ``feature`` branch. It's good practice to never work on the ``master`` branch!
+
+4. Develop the feature on your feature branch. Add changed files using ``git add`` and then ``git commit`` files:
+
+   ```bash
+   $ git add modified_files
+   $ git commit
+   ```
+
+   to record your changes in Git, then push the changes to your GitHub account with:
+
+   ```bash
+   $ git push -u origin my-feature
+   ```
+
+5. Follow [these instructions](https://help.github.com/articles/creating-a-pull-request-from-a-fork)
+to create a pull request from your fork. This will send an email to the committers.
+
+(If any of the above seems like magic to you, please look up the
+[Git documentation](https://git-scm.com/documentation) on the web, or ask a friend or another contributor for help.)
+
+Pull Request Checklist
+----------------------
+
+We recommended that your contribution complies with the
+following rules before you submit a pull request:
+
+-  Follow the
+   [coding-guidelines](http://pydicom.org/dev/developers/contributing.html#coding-guidelines).
+
+-  Use, when applicable, the validation tools and scripts in the
+   `sklearn.utils` submodule.  A list of utility routines available
+   for developers can be found in the
+   [Utilities for Developers](http://pydicom.org/dev/developers/utilities.html#developers-utils)
+   page.
+
+-  If your pull request addresses an issue, please use the pull request title to
+   describe the issue and mention the issue number in the pull request
+   description. This will make sure a link back to the original issue is
+   created.
+
+-  All public methods should have informative docstrings with sample
+   usage presented as doctests when appropriate.
+
+-  Please prefix the title of your pull request with `[MRG]` (Ready for
+   Merge), if the contribution is complete and ready for a detailed review.
+   Two core developers will review your code and change the prefix of the pull
+   request to `[MRG + 1]` and `[MRG + 2]` on approval, making it eligible
+   for merging. An incomplete contribution -- where you expect to do more work before
+   receiving a full review -- should be prefixed `[WIP]` (to indicate a work
+   in progress) and changed to `[MRG]` when it matures. WIPs may be useful
+   to: indicate you are working on something to avoid duplicated work,
+   request broad review of functionality or API, or seek collaborators.
+   WIPs often benefit from the inclusion of a
+   [task list](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments)
+   in the PR description.
+
+-  All other tests pass when everything is rebuilt from scratch. On
+   Unix-like systems, check with (from the toplevel source folder):
+
+      ```bash
+      $ make
```

Comment:
Not cython in pydicom so no Makefile. Probably we should add one to make the testing and build the doc locally. It can help new comers.